### PR TITLE
add score information for active hosts in ./siac hostdb -v

### PR DESCRIPTION
**Pull request by DavidVorick**
This PR is a PR that is helping to lead up to an overhaul of how the allowance works. I'm programming the allowance to switch away from hosts if they become uncompetitive, which requires understanding what counts as competitive.

This PR makes it a lot easier to understand how competitive the hosts are relative to eachother, given the current scoring methods.

### CODE REVIEW

File: siac/hostdbcmd.go [hunk 1]
```diff
@@ -227,9 +229,18 @@ func hostdbcmd() {
...
 			}
 
+			// Grab the score information for the active hosts.
+			hostInfo := new(api.HostdbHostsGET)
+			err := getAPI("/hostdb/hosts/"+host.PublicKeyString, hostInfo)
+			if err != nil {
+				die("Could not fetch provided host:", err)
+			}
+			score, _ := big.NewFloat(1).SetInt(hostInfo.ScoreBreakdown.Score.Big()).Float64()
+			score /= 1e70
```
Comment (lukechampine) [inline siac/hostdbcmd.go]: `1e70` seems a bit magic

Comment (DavidVorick) [inline siac/hostdbcmd.go]: It's very magic. And the worst part is, I don't know of a good way to de-magic it, except maybe to squish it with a logarithm. But then users might get confused about just how terrible a low score is.

**Comment by DavidVorick** [comment]
Ok, updated this PR to use normalization instead of blindly dividing by 1e70. The normalization uses a type of median, which means that as the hostdb changes the displayed score for a host may change even though it's absolute score has not changed. This is fine though, because the scores are purely relative anyway.

API documentation has been updated appropriately as well.

File: siac/hostdbcmd.go [hunk 2]
```diff
@@ -183,21 +184,35 @@ func hostdbcmd() {
...
+		// point.)
+		referenceScore := float64(1)
+		if len(activeHosts) > 0 {
+			referenceIndex := len(activeHosts) / 5
+			hostInfo := new(api.HostdbHostsGET)
+			err := getAPI("/hostdb/hosts/"+activeHosts[referenceIndex].PublicKeyString, hostInfo)
+			if err != nil {
+				die("Could not fetch provided host:", err)
+			}
+			referenceScore, _ = big.NewFloat(1).SetInt(hostInfo.ScoreBreakdown.Score.Big()).Float64()
```
Comment (lukechampine) [inline siac/hostdbcmd.go]: nbd, but `new(big.Float)` is our usual style

also, seems like you are converting a `big.Int` to a `big.Float` and then a `float64`, then converting other `big.Int`s to `big.Floats` and then `float64`s and then dividing them by the first `float64`. Couldn't you do the division on `big.Float`s directly? `big.Float` supports the standard printf directives, so you don't even have to change your format.

Comment (DavidVorick) [inline siac/hostdbcmd.go]: oh, I guess that works. Didn't realize that there was no need to convert to a float64.

Comment (DavidVorick) [inline siac/hostdbcmd.go]: hmm I don't see a way to div floats

**Pull request merged**